### PR TITLE
EN-5117 limit collectd process resident memory

### PR DESCRIPTION
**Pull request by sokrahta**
Paired-with: noah.meyerhans and robert.voyer

---
This change is [Reviewable](https://reviewable.io/reviews/socrata/shipyard/50)

### CODE REVIEW

File: runit/sv/collectd-run [hunk 1]
```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 sleep 10 # need collectd to start after other processes
+ulimit -m 67108864 # set resource max: 64m resident memory
```
Comment (jreichhold) [inline runit/sv/collectd-run]: Any evidence that this is enough memory? I.e. curious why this limit?

Comment (jreichhold) [inline runit/sv/collectd-run]: Also, would it be better to limit vss via -v here?

Comment (nmeyerhans) [inline runit/sv/collectd-run]: ulimit memory values are in units of kB, so this should be 65536...

However, I think @jreichhold is correct that we need to put a little more thought into the number. I just conducted an experiment in which I killed collectd in the rammy-konsument container on ip-10-140-17-30.us-west-2.compute.internal to let runit relaunch it. The freshly launched process had an RSS of 98368, so 65536 clearly isn't enough.

**Comment by jreichhold** [comment]
:shipit: can play with vss later

**Comment by sokrahta** [comment]
awesome, thanks @jreichhold

**Pull request merged**